src/import/pull-common.c

```diff
@@ -302,8 +312,12 @@ int pull_make_verification_jobs(
         if (verify == IMPORT_VERIFY_SIGNATURE) {
                 _cleanup_free_ char *signature_url = NULL;
 
-                /* Queue job for the SHA256SUMS.gpg file for the image. */
-                r = import_url_change_last_component(url, "SHA256SUMS.gpg", &signature_url);
+                /* Queue job for the signature file for the image. */
+                if (retry)
+                        r = import_url_change_last_component(url, "SHA256SUMS.gpg", &signature_url);
+                else
+                        r = import_url_change_last_component(url, chksums, &signature_url);
+
                 if (r < 0)
```

Comment:
so, this creates two jobs now for the suse case. That looks less than ideal. I am pretty sure we should simply not create a distinct signature job in the suse case, and only have the checksum job instead covering for both.